src/middleware/logger.js

```diff
@@ -6,16 +6,21 @@ module.exports = function (app) { // eslint-disable-line func-names
   // Add a logger to our app object for convenience
   app.logger = winston; // eslint-disable-line no-param-reassign
 
+  // Get log setting from config file
   const logConfig = app.get('logger');
 
+  // Let winston write logs to a file
   if (process.env.NODE_ENV !== 'test') {
+    // Create a log folder if it does not exist
     if (!fs.existsSync(logConfig.path)) {
       fs.mkdirSync(logConfig.path);
     }
+    // Set file to be written
     winston.add(winston.transports.File, {
       filename: path.join(logConfig.path, logConfig.filename),
     });
   } else {
+    // In test environment, do not print log to console
```

Comment:
It intervenes the mocha output and makes it hard to read (and annoy also). 

In `test` environment (not `dev` environment), we just want to compare the actual outcome with expected outcome; no need logging. 

Anyway, if we would like to log something out to console, we still can use console.log instead.